docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -783,29 +765,47 @@ kotlin {
     }
 
     // Configure all compilations of all targets:
-    targets.all {
-        compilations.all {
-            compilerOptions.configure {
-                allWarningsAsErrors = true
-            }
-        }
+    compilerOptions {
+        allWarningsAsErrors = true
     }
 }
 ```
 
 </tab>
 </tabs>
 
-Alternatively, to configure compiler options that are common for all targets, you can use the `compilerOptions {}` [top-level block](multiplatform-dsl-reference.md#top-level-blocks):
+## Compiler options
+
+You can configure compiler options in your projects at three different levels:
+
+* **Extension level**, by using it as a top-level block.
```

Comment:
I was a bit lost trying to understand what "it" refers to here. We could mention the `compilerOptions {}` block at least in the first list item, to smooth the experience.

Reply:
What about this instead:
* **Extension level**, as a top-level block.
* **Target level**, in the target block.
* **Compilation unit level**, usually as a specific compilation task.
Then we can keep the structure lean and explain exactly _how_ (with the `compilerOptions {}` block) later in the section. WDYT?